src/cryptoManager.ts

```diff
@@ -30,26 +36,36 @@ export class CryptoManager {
     }
   }
 
-  private async computeHash(keyFilePath: string): Promise<string> {
-    this.logger.debug('computing hash key');
-    const secret = await fsp.readFile(keyFilePath, { encoding: 'binary' });
-    const hash = crypto.createHash('sha256');
+  private async computeHash(filePath: string): Promise<Buffer> {
+    const secret = await fsp.readFile(filePath, { encoding: 'binary' });
+    const hash = crypto.createHash('sha512');
```

Comment:
move to config (no need for environment variable)

Reply:
fixed